### PR TITLE
final pieces for auth

### PR DESCRIPTION
**Pull request by des-des**
Now we are actually checking if people are from founders and coders.

Also have added username / login link in header.

### CODE REVIEW

File: templates/layout/default.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
   </head>
   <body>
       <main>
-        {{> nav}}
+        {{> nav credentails}}
```
Comment (sohilpandya) [inline templates/layout/default.html]: could you please rename this to credentials and the other places where it;s been spelt incorrectly.

**Comment by sohilpandya** [comment]
@des-des could you please update the above? so we can merge. 👍 💯

**Comment by des-des** [comment]
@sohilpandya I don't follow

**Comment by sohilpandya** [comment]
@des-des I think credentials is spelt incorrectly `credentails` or have I got that wrong?

**Comment by des-des** [comment]
@sohilpandya Ok fixed, I also push up some more bug fixes.. (sorry should have put them on a new branch)

**Pull request merged**